src/Symfony/Component/Lock/Factory.php

```diff
@@ -28,7 +28,7 @@ class Factory implements LoggerAwareInterface
 
     private $store;
 
-    public function __construct(StoreInterface $store)
+    public function __construct(PersistStoreInterface $store)
```

Comment:
Does it actually make sense to change the typehint of the deprecated class ? It will also be removed in 5.0 at the same time than the StoreInterface anyway.

Reply:
removed

Reply:
https://github.com/symfony/symfony/pull/32562